### DISCUSSION TITLE
btm with solarized light color scheme

### DISCUSSION THREAD

**Discussion by soloturn** [ClementTsang/bottom#588] [General]
could you please add a couple of lines how one could run btm so the colors are visible on light background terminal windows like solarized-light?

**Comment by ClementTsang** [comment]
You can add custom colours to btm in the [default config file](https://clementtsang.github.io/bottom/nightly/configuration/config-file/default-config/), see [this section](https://clementtsang.github.io/bottom/nightly/configuration/config-file/theming/) in the documentation.  So you could grab the colour palette for solarized-light and add them as desired. Once you add it, when you run bottom, it should use those colours instead.

In the default config file, the lines are actually already there, just commented out.  So look for something like:

```toml
#[colors] # Uncomment if you want to use custom colors
# Represents the colour of table headers (processes, CPU, disks, temperature).
#table_header_color="LightBlue"
```

And uncomment the first line and any other lines you want to set colours for:

```toml
[colors]
# Represents the colour of table headers (processes, CPU, disks, temperature).
table_header_color="Red"
```

Unfortunately this is the only real way to set colours as of now, there's no preview system or live updates.

---

There also isn't really a difference between setting colours for a light or dark background. For example, if the default text colour is unreadable on a light terminal, just set it to a darker colour.
